library/core/src/array/mod.rs

```diff
@@ -382,7 +382,7 @@ impl<T: Ord, const N: usize> Ord for [T; N] {
 
 // The Default impls cannot be done with const generics because `[T; 0]` doesn't
 // require Default to be implemented, and having different impl blocks for
-// different numbers isn't supported yet.
+// different numbers aren't supported yet.
```

Comment:
```suggestion
// different numbers isn't supported yet.
```

This should be `isn't`: `having ~ isn't supported yet.`

Reply:
Oh whoops my bad